OMCompiler/SimulationRuntime/c/util/simulation_options.c

```diff
@@ -257,6 +268,17 @@ const char *FLAG_DESC[FLAG_MAX+1] = {
   /* FLAG_R */                            "value specifies a new result file than the default Model_res.mat",
   /* FLAG_DATA_RECONCILE */               "Run the Data Reconciliation numerical computation algorithm for constrained equations",
   /* FLAG_DATA_RECONCILE_BOUNDARY */      "Run the Data Reconciliation numerical computation algorithm for boundary condition equations",
+  /* FLAG_SR */                           "value specifies the chosen outer ODE solver (adams)",
```

Comment:
Doc string not clear. It is not clear what an outer ODE solver (adams) should be without knowing, that this flag is related to the GBODE method. Same for the other strings.

Maybe do something like:

```C
  /* FLAG_SR */                           "Single rate solver method for GBODE integrator method.",
```